AccountService/appsettings.example.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
AccountService/appsettings.example.json rare naam,verteld niet veel verwijderen als nodig is?